src/zarr/core/group.py

```diff
@@ -1067,26 +1066,26 @@ async def create_array(
             List of compressors to apply to the array. Compressors are applied in order, and after any
             filters are applied (if any are specified) and the data is serialized into bytes.
 
-            For Zarr format 3, a "compressor" is a codec that takes a bytestream, and
+            For Zarr format 3, a "compressor" is a codec that takes a bytestrea, and
```

Comment:
```suggestion
            For Zarr format 3, a "compressor" is a codec that takes a bytestream, and
```

Reply:
This resurfaced due to merge conflicts that I resolved.